cli/src/commands/extensions/api.rs

```diff
@@ -100,6 +100,7 @@ impl From<ProcessException> for permissions::Permissions {
             run: process_exception.run,
             env: process_exception.env,
             net: Permission::Boolean(process_exception.net),
+            unsandboxed_run: Permission::default(),
```

Comment:
I assume this would default to `false`; why is that? Wouldn't we rather want to augment `ProcessException` here?

Reply:
I'm not sure what you mean? `ProcessException` only applies to `runSandboxed` and as such `unsandboxed_run` should have no impact on it?